frb_example/with_flutter/lib/bridge_generated.io.dart

```diff
@@ -12,7 +12,26 @@ import 'package:meta/meta.dart';
 import 'dart:ffi' as ffi;
 
 class FlutterRustBridgeExamplePlatform extends FlutterRustBridgeBase<FlutterRustBridgeExampleWire> {
-  FlutterRustBridgeExamplePlatform(ffi.DynamicLibrary dylib) : super(FlutterRustBridgeExampleWire(dylib));
+  final _port = RawReceivePort();
+  NativePortType get port => _port.sendPort.nativePort;
+  FlutterRustBridgeExamplePlatform(ffi.DynamicLibrary dylib) : super(FlutterRustBridgeExampleWire(dylib)) {
+    var initResult =
```

Comment:
What if there are two `FlutterRustBridgeExamplePlatform` instances (or two instances of different such auto-generated classes of frb)? Will we face trouble because we init twice?

Moreover, consider hot-restart scenario: Even if users have only *one* such instance, it will be created multiple times (and without dispose at all)

Btw, do we really need it in Flutter scenarios? Since flutter should already init dart

Reply:
from dart_api_dl.h
 ```
 * All symbols are postfixed with _DL to indicate that they are dynamically
 * linked and to prevent conflicts with the original symbol.
 *
 * Link `dart_api_dl.c` file into your library and invoke
 * `Dart_InitializeApiDL` with `NativeApi.initializeApiDLData`.
 ```
 made it so that initialization is called once

Reply:
I see. Just a remark that, the `FlutterRustBridgeExamplePlatform` constructor can be called *multiple times* because of the following scenarios for example:

1. hot restart flutter app
2. users (for whatever reason) create two FlutterRustBridgeExamplePlatform instances

Reply:
one-time static initialization

Reply:
Have you tried the hot-restart-flutter case, because in that case, it is initialized multiple times (even though we use static variables etc), because after each hot restart, the Dart vm starts freshly, while state in native code is kept. So I am not sure whether that is OK.

Reply:
I tested and didn't get multiple initialization